src-docs/src/views/datagrid/datagrid_cell_popover_example.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import { GuideSectionTypes } from '../../components';
+import { EuiCode } from '../../../../src/components';
+
+import IsDetailsPopover from './cell_popover_is_details';
+const IsDetailsPopoverSource = require('!!raw-loader!./cell_popover_is_details');
+
+import RenderCellPopover from './cell_popover_rendercellpopover';
+const renderCellPopoverSource = require('!!raw-loader!./cell_popover_rendercellpopover');
+
+import IsExpandablePopover from './cell_popover_is_expandable';
+const isExpandablePopoverSource = require('!!raw-loader!./cell_popover_is_expandable');
+
+import {
+  EuiDataGridCellPopoverElementProps,
+  EuiDataGridCellValueElementProps,
+  EuiDataGridColumn,
+} from '!!prop-loader!../../../../src/components/datagrid/data_grid_types';
+
+export const DataGridCellPopoverExample = {
+  title: 'Data grid cell popovers',
+  sections: [
+    {
+      title: 'Conditionally customizing cell popover content',
+      text: (
+        <>
+          <p>
+            Cell popover content values can be conditionally customized using
+            the <EuiCode>isDetails</EuiCode> flag passed to{' '}
+            <EuiCode>renderCellValue</EuiCode>. If you need basic customization
+            of cell popover values based on, e.g. schema or column, this is the
+            most straightforward approach.
+          </p>
+          <p>
+            By default, all cell popover contents are rendered with an{' '}
+            <strong>EuiText</strong> wrapper, and cell actions are rendered
+            within an <strong>EuiPopoverFooter</strong> as{' '}
+            <strong>EuiEmptyButton</strong>s. Columns with a{' '}
+            <EuiCode>json</EuiCode> schema will additionally have an automatic
+            formatter that indents and displays the popover content within an{' '}
+            <strong>EuiCodeBlock</strong>.
+          </p>
+        </>
+      ),
+      demo: <IsDetailsPopover />,
+      components: { IsDetailsPopover },
+      source: [
+        {
+          type: GuideSectionTypes.TSX,
+          code: IsDetailsPopoverSource,
+        },
+      ],
+      props: {
+        EuiDataGridCellValueElementProps,
+      },
+    },
+    {
+      title: 'Completely customizing cell popover rendering',
+      text: (
+        <>
+          <p>
+            If you want complete control over the rendering of the entire cell
+            popover, use the <EuiCode>renderCellPopover</EuiCode> prop to pass a
+            functional component. This allows you to do things like set your own
```

Comment:
```suggestion
            component. This allows you to do things like set your own
```

Any component will do

Reply:
https://github.com/elastic/eui/pull/5640/commits/96a4bf1a2c12a5e03f0301c82770b6bf187e34a6